src/site/content/en/blog/badging-api/index.md

```diff
@@ -158,9 +153,52 @@ Android only ever shows a dot instead of a numeric value.
 Don't assume anything about how the user agent wants to display the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
-then the "+" won't appear. No matter the actual number, just set
-`Badge.set(unreadCount)` and let the user agent deal with displaying it
-accordingly.
+then the "+" won't appear. No matter the actual number, just call
+`setExperimentalAppBadge(unreadCount)` and let the user agent deal with
+displaying it accordingly.
+
+While the Badging API *in Chrome* requires an installed app with an icon
+that can be badged, you shouldn't make calls to the Badging API dependent
```

Comment:
What's an icon that can be badged?

Reply:
Just drop "with an icon that can be badged".